src/main/java/skeltal/TaskList.java

```diff
@@ -0,0 +1,91 @@
+package skeltal;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    /**
+     * A method to add a Task object into the list.
+     * @param assignment A task object.
+     */
+    public static void addReply(Task assignment) {
```

Comment:
Any reason why it's named `addReply`? It seems like you're adding a reply rather than adding a task 😅